Automattic/jetpack
# AI Assistant: Fix crash when transforming to AI Assistant block

Closes #31691 

## Proposed changes:

* Add `@wordpress/block-editor` types.
* Move `getTextContentFromSelectedBlocks ` to be an hook (`useTextContentFromSelectedBlocks `).
* Remove unused blocks when transforming to `AI Assistant`.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Add paragraphs in your post.
* Select multiple (2, 3, 4 ...).
* Convert them using `Ask Ai Assistant` button.
* They should be all converted and current blocks removed.
